package.json

```diff
@@ -2341,5 +2343,14 @@
     },
     "optionalDependencies": {
         "canvas": "^2.7.0"
+    },
+    "resolutions": {
```

Comment:
List of packages that need to be pinned to specific versions.
Once we move to npm 8.0 this will not be necessary, npm 8 supports this sort of feature natively

Reply:
Interesting. I knew that yarn didn't this, didn't know there was an npm package to do the same thing.